bblfsh/client.py

```diff
@@ -1,54 +1,57 @@
 import os
-import sys
+import typing as t
```

Comment:
Let's import individual types from `typing`. `t` is too short and error-prone while `typing` is too long for complex type defs.